### PR TITLE
Add a preloader to Onelive & IRC pages

### LINKED ISSUE #1082: Add a preloader to Onelive & IRC pages
**Is your feature request related to a problem? Please describe.**
The Onelive & IRC pages are taking some time to load the data to the pages. It shows as in below for few seconds

Screenshot from 2021-08-15 10-18-23
Screenshot from 2021-08-15 10-18-37

**Describe the solution you'd like**
Add a preloader to Onelive & IRC pages to avoid that 
* Take a look at the PR for the Team page preloader (https://github.com/sef-global/sef-site/pull/1077)
* Already the CSS is created for the preloader
* Add the related content to the HTML files

### PR DESCRIPTION
**Pull request by fustroli**
## Purpose

The purpose of this PR is to fix #1082

## Goals

Add skeleton loading profiles until the profiles are loaded

## Approach

Used skeleton loading profiles in the relevant sections as in #1077

### Screenshots

Screenshot from 2021-08-16 12-22-40
Screenshot from 2021-08-16 12-23-47

  
### Preview Link

https://pr-1087-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
 
#1077 

## Test environment
 
Linux, Brave, Chrome

### CODE REVIEW

File: irc.html [hunk 1]
```diff
@@ -395,4 +453,4 @@ <h5 class="title">
       </table>
 </script>
 
-</html>
+</html>
```
Comment (Gravewalker666) [inline irc.html]: EOF

File: irc.html [hunk 2]
```diff
@@ -64,7 +64,65 @@ <h1 class="display-2 pb-3" style="line-height: 1.2;">International Research Coun
...
+                            <div class="row profile-name ">
+                                <div class="line"></div>
+                                <div class="line line-secondary"></div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        </div>
```
Comment (Gravewalker666) [inline irc.html]: Is this dev properly closed??

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]

File: irc.html (hunk 2 above)
Comment (fustroli) [reply irc.html]: Yes, just some indentation problem. Fixed it.

File: irc.html (hunk 1 above)
Comment (fustroli) [reply irc.html]: Fixed it.

**Comment by fustroli** [comment]
@Gravewalker666  I tried to squash the commit but was unable to. Followed the article in the Contributors guide, got a "Successfully rebased and updated" message but there are still two commits. Don't know how to proceed.

**Comment by Gravewalker666** [comment]
> I tried to squash the commit but was unable to. Followed the article in the Contributors guide, got a "Successfully rebased and updated" message but there are still two commits. Don't know how to proceed.

Not to worry my friend, We'll squash them on the merge :)

File: onelive/index.html [hunk 3]
```diff
@@ -94,12 +94,71 @@ <h1 class="display-3 text-center mb-5">Take a look at our past OneLives</h1>
                     <h1 class="display-3 text-center mb-5">Speakers</h1>
                 </div>
             </div>
-            <div class="row justify-content-center" id="btnSpeakers" style="visibility:hidden;">
+            <div  >
```
Comment (janethavi) [inline onelive/index.html]: Shall we remove the extra space on the `<div>` tag?

Comment (fustroli) [reply onelive/index.html]: Fixed

**Review by janethavi — APPROVED** [review]
Great Work @VWRoli

**Review by Gravewalker666 — APPROVED** [review]
Awesome job @VWRoli

**Pull request merged**